src/Cosmos/Protobuf/thorchain_bank_tx.proto

```diff
@@ -6,9 +6,33 @@ package types;
 
 import "coin.proto";
 
+// Original: https://gitlab.com/thorchain/thornode/-/blob/develop/proto/thorchain/v1/common/common.proto
+message Asset {
```

Comment:
I guess you could use `src/proto` directly 🤔?

Reply:
Cosmos has two signing modes: JSON and Protobuf. If Protobuf signing mode is chosen, we convert `Cosmos::SigningInput` message into a Cosmos native protobuf messages, serialize them and sign a hash.
So `src/Cosmos/Protobuf/` directory contains the Cosmos native protobuf messages for internal use only, not for the public WalletCore interface